pkg/component/observability/monitoring/prometheus/garden/testdata/seed.prometheusrule.test.yaml

```diff
@@ -0,0 +1,31 @@
+rule_files:
+- seed.prometheusrule.yaml
+
+tests:
+  - name: SeedExtensionsNotReady:SmoothOutFlapping
+    interval: 1m
+    input_series:
+      - series: garden_seed_condition{condition = "ExtensionsReady",
+                                      name      = "seed-flapping"}
+        values: 0x2 2 0 -1 2 0x2 2 0
```

Comment:
Maybe if the repetition is so low, we could also resolve the pattern for simplicity:


```
0x2 2 0 -1 2 0x2 2 0
```
-->
```
0 0 2 0 -1 2 0 0 2 0
```

Maybe for readability, we could mention in a comment here how the states are mapped to numbers by the gardener-metrics-exporter. https://github.com/gardener/gardener-metrics-exporter/blob/436fb84907a3778b0e90287155fe32b676942d2c/pkg/metrics/utils.go#L28-L39

> 2: Progressing
> 1: True
> 0: False
> -1: Unknown

Reply:
I addressed this by mentioning it in the first commit adding a promtool test: [08f99a4](https://github.com/gardener/gardener/pull/9750/commits/08f99a40a1c470c526299752584ca5fa51017ee7) so it helps when going through the whole commit history.

Reply:
As discussed off line, maybe add this information as a comment to the yaml test file as well, to improve the readability of the test for humans by resolving the "magic numbers" to concepts like Progressing/True/False/Unknown.